axelor-account/src/main/resources/views/AccountingBatch.xml

```diff
@@ -163,7 +163,7 @@
         <field name="moveDescription" requiredIf="actionSelect == 26"/>
         <panel colSpan="6" itemSpan="12">
           <field name="moveDate" requiredIf="actionSelect == 26"
-            validIf="$moment(moveDate).date() == $moment(moveDate).endOf('month').date()"
+            validIf="actionSelect != 26 || $moment(moveDate).date() == $moment(moveDate).endOf('month').date()"
```

Comment:
is this change wanted ? it seems that it is not related to the issue.

Reply:
Yes, this fix is to be on wip-6.4 so can be done here